room/compiler-processing-testing/src/main/java/androidx/room/compiler/processing/util/ProcessorTestExt.kt

```diff
@@ -115,12 +115,14 @@ fun runProcessorTest(
     sources: List<Source> = emptyList(),
     classpath: List<File> = emptyList(),
     options: Map<String, String> = emptyMap(),
+    compilerArguments: (CompilationType) -> List<String> = { emptyList() },
     handler: (XTestInvocation) -> Unit
 ) = runProcessorTest(
     sources = sources,
     classpath = classpath,
     options = options,
-    handlers = listOf(handler)
+    handlers = listOf(handler),
+    compilerArguments = compilerArguments
```

Comment:
nit: lets keep handlers the last argument

Reply:
well not fully sure actually because it is the only function type parameter here.
though i'm thinking, maybe we should use an explicit type here, or fun interface? specifically, for other overloads that receive a handler block, idk how well IJ will resolve it.

Reply:
also, i think this is not used in this PR? do you need it on your end?

Reply:
I don't actually need this anymore, I guess I will just delete it unless you think it would be useful to keep. I added it because I thought it might be necessary to enable experiment flags like inline classes or to enable multiplatform, but that didn't pan out in this PR